src/modules/backlight.c

```diff
@@ -151,7 +151,9 @@ static void set_keyboard_level(const double level) {
          * on low ambient brightness, it must be turned on
          */
         state.current_kbd_pct = 1.0 - level;
-        if (call(NULL, NULL, &kbd_args, "i", state.current_kbd_pct * max_kbd_backlight) == 0) {
+	double new_value = state.current_kbd_pct * max_kbd_backlight;
+	new_value += 0.5 - (new_value<0); // rounding
```

Comment:
Indentation and spacing

Reply:
Btw what is this needed for? I am just curious!

Reply:
if you mean the new_value variable, it's just for rounding to the closest int, so it won't just cut all decimals when converting to int. As the command for changing brightness takes only int values. It's just nice to have imo.

Reply:
Ok, I see. I meant, i understood the simple algorithm; it was more about why not using [round](https://linux.die.net/man/3/round) instead. 
I did not understand if there was any reason behind that!

Reply:
Oh, I tried to make it so we won't require math.h. Didn't notice that it was already included. I will fix it.